install.sh

```diff
@@ -194,9 +190,41 @@ __install_bro() {
 	__install_packages bro broctl
 	chmod 2755 /opt/bro/logs
 	_BRO_PKG_INSTALLED=true
+	_BRO_INSTALLED=true
 	_BRO_PATH="/opt/bro/bin"
 }
 
+__configure_bro() {
+	# Configure Bro
+	curl -sSL "https://raw.githubusercontent.com/activecm/bro-install/master/gen-node-cfg.sh" -o "./gen-node-cfg"
```

Comment:
gen-node-cfg is set up for the bro-install repo which uses pf ring. Unfortunately this causes issues when users are using a standard installation of bro ids. Namely, it sets the option `lb_method=pf_ring`.

Reply:
We've found in practice that `lb_method=pf_ring` doesn't have any effect if pf_ring isn't available.  So essentially it helps a ton if it happens to be there (not likely) but doesn't seem to do any harm if it isn't.  Even when pf_ring isn't available the config sets up a number of workers and processes that helps spread the workload efficiently across the CPU.  This is what every AI-Hunter install uses.

Reply:
Additionally ran into an issue where the interface I wanted to use as a monitor was down. I had left it down since I figured the script would take care of everything. However, this prevented me from choosing the interface as a sniffing interface in gen-node-cfg. I created the necessary changes in /etc/network/interfaces to bring the interface up without an IP address and opened an issue in the bro-install repo to address this use case.